dpgen2/utils/run_command.py

```diff
@@ -1,13 +1,13 @@
 from dflow.utils import run_command as dflow_run_command
 from typing import Tuple, Union, List
+import os
 
 
 def run_command(
     cmd: Union[str, List[str]],
     shell: bool = False,
 ) -> Tuple[int, str, str]:
+    interactive = False if os.getenv("DFLOW_DEBUG") else True
```

Comment:
getting the value from `dflow.config["mode"]` would be better? 

In an extreme case, the user changes the value of `DFLOW_DEBUG` as the workflow runs, the assignment would be problematic.